ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/telephoneinput/v1/telephoneinput/clientlibs/site/js/telephoneinputview.js

```diff
@@ -78,6 +79,26 @@
                 this.setActive();
             });
         }
+
+        /**
+         * Binds Telephone input event handling for number input only
+         *
+         * @private
+         */
+        #bindEvents(telephoneInput) {
```

Comment:
Why are we preventing alpha-numeric input? Won't phone number be of format like +91 9999999 ?